### DISCUSSION TITLE
snackbar/modal manager

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#6431] [Ideas] [answered]
Не нашел в документации упоминаний единого менеджера нотификаций/уведомлений. Мне кажется полезная и удобная фича, которая упростит использование модальных окон и снекбаров. Планируете ли внедрять что-то похожее? Может быть есть какие-то подводные камни? 

Референсы:
* mantine [notifications](https://mantine.dev/x/notifications/) / [modals](https://mantine.dev/x/modals/)
* ant [notifications](https://ant.design/components/notification) / [modals](https://ant.design/components/modal#modalmethod)

**Comment by inomdzhon** [comment, marked as answer]
Привет,

Идею, если что, обсуждаем. По Snackbar уже делали протик https://github.com/VKCOM/VKUI/pull/8330.

**Comment by EldarMuhamethanov** [comment]
Создал два черновика pull request с примерной API для менеджера снекбаров(https://github.com/VKCOM/VKUI/pull/8839) и менеджера модалок(https://github.com/VKCOM/VKUI/pull/8847)
